knative-operator/pkg/common/eventing.go

```diff
@@ -1,45 +0,0 @@
-package common
-
-import (
-	"os"
-
-	"k8s.io/apimachinery/pkg/api/resource"
-
-	operatorv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
-)
-
-func MutateEventing(ke *operatorv1alpha1.KnativeEventing) {
-	eventingImagesFromEnviron(ke)
-	ensureEventingWebhookMemoryLimit(ke)
-	ensureEventingWebhookInclusionMode(ke)
-	defaultToEventingHa(ke)
-}
-
-func defaultToEventingHa(ke *operatorv1alpha1.KnativeEventing) {
-	if ke.Spec.HighAvailability == nil {
-		ke.Spec.HighAvailability = &operatorv1alpha1.HighAvailability{
-			Replicas: 2,
```

Comment:
Ah, so we get rid of this duplicaton, since we really wanna do it here:
https://github.com/openshift-knative/serverless-operator/blob/5454c1798b236e613daeb3a2629038258fef9416/openshift-knative-operator/pkg/eventing/extension.go#L63-L68

Is that the motivation here?

Reply:
Partially. HA setup is one spot where we might think about **actually** keep the stamping, as it feels kinda sensible. For other parts though (default domains, image overrides etc.) it doesn't make any sense at all, as those pieces are supposed to change with version and cluster environment.

Reply:
but on the extension also set memory, b/c the openshift needs a bit more than upstream.

This is also done in the "to be removed" file: https://github.com/openshift-knative/serverless-operator/blob/94fa6bb25e9ddff08728f1b7fab20f9701407397/knative-operator/pkg/common/eventing.go#L38

and the extension: https://github.com/openshift-knative/serverless-operator/blob/5454c1798b236e613daeb3a2629038258fef9416/openshift-knative-operator/pkg/eventing/extension.go#L56


> HA setup is one spot where we might think about actually keep the stamping
do you want to remove more of the "overrides" we are currently doing (in the extension)? 

sorry, if I miss something here

Reply:
We're not removing any overrides in the extension.